code/modules/paperwork/paper.dm

```diff
@@ -620,43 +620,76 @@ var/global/list/paper_folding_results = list ( \
-						</fieldset><BR>
-						Name: [identity]<BR>
-						Blood Type: [merchant.dna.b_type]<BR>
-						Fingerprint: [md5(merchant.dna.uni_identity)]</body></html>"}
-		display_y = 700
-		CentcommStamp(src)
+		apply_text(merchant)
 	..()
 
-/obj/item/weapon/paper/merchantreport/show_text(var/mob/user, var/links = FALSE, var/starred = FALSE)
+/obj/item/weapon/paper/merchant/show_text(var/mob/user, var/links = FALSE, var/starred = FALSE)
 	var/index = 1
 	for(var/image in mugshots)
 		user << browse_rsc(image, "previewicon-[identity][index].png")
 		index++
 	..()
 
+/obj/item/weapon/paper/merchant/proc/apply_text(mob/living/carbon/human/merchant)
+	identity = merchant.client.prefs.real_name
+	icon = 'icons/obj/items.dmi'
+	icon_state = "permit"
+	name = "Merchant's Licence - [identity]"
+	info = {"<html><style>
+			body {color: #000000; background: #ffff0d;}
+			h1 {color: #000000; font-size:30px;}
+			fieldset {width:140px;}
+			</style>
+			<body>
+			<center><img src="http://ss13.moe/wiki/images/1/17/NanoTrasen_Logo.png"> <h1>Merchant's Licence</h1></center>
+			Nanotrasen\'s commercial arm has authorized commercial activity for a merchant who holds a licence for corporate commerce, a process which includes a background check and Nanotrasen loyalty implant. The associate\'s image is displayed below.<BR>
+			<fieldset>
+	  		<legend>Picture</legend>
+			<center><img src="previewicon-[identity]1.png" width="64" height="64"><img src="previewicon-[identity]2.png" width="64" height="64"></center>
+			</fieldset><BR>
+			Name: [identity]<BR>
+			Blood Type: [merchant.dna.b_type]<BR>
+			Fingerprint: [md5(merchant.dna.uni_identity)]</body></html>"}
+
+/obj/item/weapon/paper/merchant/report
+	icon_updates = TRUE
+	display_y = 700
+
+/obj/item/weapon/paper/merchant/report/apply_text(mob/living/carbon/human/merchant)
+	identity = merchant.client.prefs.real_name
+	name = "Licensed Merchant Report - [identity]"
+	info = {"<html><style>
+			body {color: #000000; background: #ccffff;}
+			h1 {color: #000000; font-size:30px;}
+			fieldset {width:140px;}
+			</style>
+			<body>
+			<center><img src="http://ss13.moe/wiki/images/1/17/NanoTrasen_Logo.png"> <h1>ATTN: Internal Affairs</h1></center>
+			Nanotrasen\'s commercial arm has noted the presence of a registered merchant who holds a licence for corporate commerce, a process which includes a background check and Nanotrasen loyalty implant. The associate\'s image is enclosed. Please continue to monitor trade on an ongoing basis such that Nanotrasen can maintain highest standard small business enterprise (SBE) partners.<BR>
+			<fieldset>
+	  		<legend>Picture</legend>
+			<center><img src="previewicon-[identity]1.png" width="64" height="64"><img src="previewicon-[identity]2.png" width="64" height="64"></center>
+			</fieldset><BR>
+			Name: [identity]<BR>
+			Blood Type: [merchant.dna.b_type]<BR>
+			Fingerprint: [md5(merchant.dna.uni_identity)]</body></html>"}
```

Comment:
If the text is going to be largely the same across these, you might consider making this an inherited function that just renders from a template.

Reply:
While the bottom part is the same, the text is subtly different and the background color is different too. If it was the whole thing I'd agree with you.